spec/spec.md

```diff
@@ -0,0 +1,2604 @@
+---
```

Comment:
Seems like the author list doesn't render well

Reply:
As far as I understand it, this is the correct markdown syntax for specifying an author (if you want to convert markdown to something else).  If I run this through pandoc (`pandoc spec.md --toc -N -o spec-md.pdf`), I get something almost exactly like the original:

<img width="644" alt="Screen Shot 2021-11-13 at 11 15 22" src="https://user-images.githubusercontent.com/1018530/141650990-ebea0f42-d0f4-437c-b25e-6f1b80078ecc.png">

Reply:
With the newest version, this should render exactly as before. 🎊 

<img width="919" alt="Screen Shot 2022-02-16 at 01 33 35" src="https://user-images.githubusercontent.com/1018530/154209375-82ed9fcc-733f-4d69-915d-6a4ada3e7901.png">